removed relative paths from having base dir added

ephur opened the pull request:
Also catch exceptions and print warnings, since sometiems it may be desirable for molecule to not write to a file

rgreinho commented on molecule/utilities.py:
```diff
@@ -90,8 +90,11 @@ def write_file(filename, content):
+        with open(filename, 'w') as f:
+            f.write(content)
+    except Exception as e:
+        raise
```
What the point of catching the exception to re-raise it right away?

ephur replied:
Force of habit -- will remove this try/catch.

rgreinho commented on molecule/provisioners.py:
```diff
@@ -168,12 +171,17 @@ def converge(self, idempotent=False):
         try:
+            print "Executing ansible_playbook"
+            print "Playbook: %s\nArgs%s\nkwargs%s" % (playbook, args, kwargs)
+            # kwargs['inventory_file'] = '../../inventory_fusion'
```
Remove comment.

rgreinho commented on molecule/config.py:
```diff
@@ -94,13 +94,15 @@ def build_easy_paths(self):
-                                                             self.config['ansible']['config_file'])
-        self.config['ansible']['inventory_file'] = os.path.join(self.config['molecule']['molecule_dir'],
-                                                                self.config['ansible']['inventory_file'])
+        vals = [
```
Can we name ``vals`` to something else? At least ``values``?

rgreinho commented on molecule/config.py:
```diff
@@ -94,13 +94,15 @@ def build_easy_paths(self):
+            ('ansible', 'inventory_file')
+        ]
+
+        for k, v in vals:
```
In that case it is not ``key`` and ``value`` anymore, so I would replace ``k`` and ``v`` with more meaningful variable names.

ephur commented:
Addressed the comments above. On the variable naming for key/value I changed it to section and item, because it reminds me of ini file format, where molecule or Ansible is a section, and config_file is an item.  Also I had the try/catch because often times in the past I've caught the list of excpetions I am 'expecting' and raise exceptions of a different type, so I am just used to that pattern but it certainly didn't add any value here. Removed.

retr0h commented:
+ 1

rgreinho commented:
:+1:
